File: GTTS/INL.PersonService.Models/GetMatchingPersons_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public class GetMatchingPersons_Param : IGetMatchingPersons_Param
    {
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public DateTime? DOB { get; set; }
        public int? POBCityID { get; set; }
        public char? Gender { get; set; }
        public string NationalID { get; set; }
        public bool? ExactMatch { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-matching-persons_param.ts


import { IGetMatchingPersons_Param } from './iget-matching-persons_param';

export class GetMatchingPersons_Param implements IGetMatchingPersons_Param {
  
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOB?: Date;
	public POBCityID?: number;
	public Gender?: string;
	public NationalID: string = "";
	public ExactMatch?: boolean;
  
}


